sympy/functions/special/bessel.py

```diff
@@ -330,6 +330,12 @@ def eval(cls, nu, z):
                 return S.NaN
         if z in (S.Infinity, S.NegativeInfinity):
             return S.Zero
+        if abs(z) is S.Infinity:
+            if im(z) > 0:
```

Comment:
The expression was derived for the limit at `I*oo`. I don't think that it would work for other infinities with positive imaginary part.

Reply:
True i checked this , it won't be true in all cases . I thought our analysis can be extended to all positive im(z) but for now let's stick to `I*oo` and `-I*oo`